conf/nginx/bedrock.conf.erb

```diff
@@ -0,0 +1,20 @@
+root /app/<%= ENV['DOCUMENT_ROOT'] %>;
+
+fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+index index.php index.html;
+
+location / {
+    try_files $uri $uri/ /index.php?$query_string;
+}
+location ~* ^.+.(jpg|jpeg|gif|css|png|js|ico|ttf|woff)$ {
+    access_log off;
+}
+location /. {
```

Comment:
Sorry these should not have been part or the pull request - I should move them into another branch for my personal use. I forgot that all the changes I made to master would be added to the previous pull request.